android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip
```

Comment:
Is really necessary to update this? This will force us to do changes on the panthalassa interface for Android and maybe impact the writing/reading direct from Real on Native side